install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1425,26 +1332,41 @@ and overwrites local changes to the metrics configurations.
 For example, the number of instances in a replica set is not saved.
 ====
 
-To update, follow the same steps as when the metrics components were
+To upgrade cluster metrics:
+
+. If you are upgrading from {product-title}
+ifdef::openshift-origin[]
+1.3 to 1.4,
+endif::[]
+ifdef::openshift-enterprise[]
+3.3 to 3.4,
+endif::[]
+first add the `view` permission to the *hawkular* service account:
++
+----
+$ oadm policy add-role-to-user view \
+        system:serviceaccount:openshift-infra:hawkular \
+        -n openshift-infra
+----
+
+. Then, follow the same steps as when the metrics components were
 xref:../../install_config/cluster_metrics.adoc#deploying-the-metrics-components[first deployed],
 using the
 xref:../../install_config/cluster_metrics.adoc#modifying-the-deployer-template[correct template],
 except this time, specify the `MODE=refresh` option:
-
-====
++
 ----
-$ oc new-app -f metrics-deployer.yaml \
+$ oc new-app --as=system:serviceaccount:openshift-infra:metrics-deployer \
+    -f metrics-deployer.yaml \
     -p HAWKULAR_METRICS_HOSTNAME=hm.example.com \
     -p MODE=refresh <1>
 ----
 <1> In the original deployment command, there was no `MODE=refresh`.
```

Comment:
@mwringe Does `--as=system:serviceaccount:openshift-infra:metrics-deployer` belong here in the metrics upgrade command, like it was added to the metrics install doc per https://github.com/openshift/openshift-docs/pull/3018? I've gone ahead and added it for now.

Please see also the first step added above (about the `view` role for the hawkular SA), per QE feedback in https://bugzilla.redhat.com/show_bug.cgi?id=1383278#c5. 

Preview build:

http://file.rdu.redhat.com/~adellape/120216/upgrade34/install_config/upgrading/manual_upgrades.html#manual-upgrading-cluster-metrics